spec/features/saml/saml_spec.rb

```diff
@@ -199,6 +199,8 @@
         click_on 'Yes'
 
         complete_idv_profile_ok(user.reload)
+        click_submit_default
+        enter_correct_otp_code_for_user(user)
```

Comment:
The `(555) 555-5555` phone number is a reserved value in the Proofer gem mock Vendor.

https://github.com/18F/identity-proofer-gem/blob/master/lib/proofer/vendor/mock.rb#L32

It's a number that will be considered "good" during the confirmation step.